doc/topics/dependencies.md

```diff
@@ -0,0 +1,33 @@
+| [libwebp](https://github.com/webmproject/libwebp)                                                                   | Decodes WebP images.                                                                                     |
+| [modp_b64](https://github.com/chromium/chromium/tree/15996b5d2322b634f4197447b10289bddc2b0b32/third_party/modp_b64) | Decodes and encodes base64.                                                                              |
+| [OpenSSL](https://github.com/openssl/openssl)                                                                       | Used to generate unique authorization tokens for authenticating with Cesium ion.                         |
+| [PicoSHA2](https://okdshin/PicoSHA2)                                                                                | Generates SHA256 hashes for use with the Cesium ion REST API.                                            |
+| [RapidJSON](https://github.com/Tencent/rapidjson)                                                                   | For JSON reading and writing.                                                                            |
+| [s2geometry](https://github.com/google/s2geometry)                                                                  | Spatial indexing library designed for geospatial use and required by some tilesets.                      |
+| [spdlog](https://github.com/gabime/spdlog)                                                                          | Debug logging.                                                                                           |
+| [sqlite3](https://www.sqlite.org/index.html)                                                                        | Used to cache HTTP responses.                                                                            |
+| [stb_image](https://github.com/nothings/stb/blob/master/stb_image.h)                                                | A simple image loader.                                                                                   |
+| [tinyxml2](https://github.com/leethomason/tinyxml2)                                                                 | XML parser for interacting with XML APIs such as those implementing the Web Map Service standard.        |
+| [uriparser](https://github.com/uriparser/uriparser)                                                                 | Used to parse and manipulate URIs.                                                                       |
+| [zlib-ng](https://github.com/zlib-ng/zlib-ng)                                                                       | An optimized zlib implementation for working with Gzipped data.                                          |
+
+The following chart illustrates the connections between the Cesium Native libraries and third-party dependencies:
+
+\image html dependency-graph.svg
```

Comment:
I can make it larger, which I've done by adding an alias that gives the viewport the specified min-height. For width, though, it's a bit trickier since it's currently limited by the width of the page container. We can get around this by adding a negative margin to the left and right sides, but this will make the pane too large on small screens. Media queries can help with this, but they can't get around the fact that the width of the sidebar can be changed, which means some JavaScript would be needed to adjust the size as it changes. I can add this JavaScript if it seems useful.